### PR TITLE
Octavia DT with networking

### PR DESCRIPTION
**Pull request by beagles**
This PR contains DT descriptions and CR customizations to enable Octavia. Note that it includes some lib changes to meet the node configuration requirements. This likely supersedes https://github.com/openstack-k8s-operators/architecture/pull/55, but discussions are ongoing.

### CODE REVIEW

File: examples/dt/VANS/octavia_default/README.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+| Keystone  | default                    | Must have         |
+| OVN       | octavia NIC mapping        | Must have         |
+| Nova      | default                    | Must have         |
+| Redis     | default                    | Must have         |
+
+### Additional information
+
+Octavia is disabled by default in the openstack operator so it must be enabled,
+with the appropriate network attachments configured to deploy Octavia. Also
+`active-standby` is enabled by setting the `loadbalancer_topology`
```
Comment (omersch381) [inline examples/dt/VANS/octavia_default/README.md:35]: I would remove the "active-standby" part from this (octavia_default) readme.

**Review by omersch381 — COMMENTED** [review]
Thanks Brent!

**Comment by fultonj** [comment]
Can you rebase this so we can see the `kustomize build` it tested now that we have PR175 merged?

https://github.com/openstack-k8s-operators/architecture/pull/175

**Comment by beagles** [comment]
Currently set to draft mode. Unified Alpha incorporated some of the changes. I'll revise the PR to target octavia related bits to lib in the event we want to enable it in other DTs/VAs.

**Pull request closed without merging**